### DISCUSSION TITLE
Data Importer behind reverse proxy does not generate correct links

### DISCUSSION THREAD

**Discussion by arn4** [firefly-iii/firefly-iii#7418] [General]
Hello! 

I'm trying to install the Data Importer for the first time, with docker compose.
I'm running it behind a reverse proxy forcing https connection with nginx, the vanity url is something like https://importer.my-firefly.net
I can access the initial page of the importer, but the problem is that all CSS and script files cannot be loaded since all the links in the returned HTML code are of the type http://importer.my-firefly.net/... All the browser are preventing the loading since they say is not secure to load external resources with HTTP if the page is instead HTTPS.

I'm quite sure the configuration of nginx is correct since I'm using the exact same configuration with the Firefly app and everything is working perfectly. The obvious solution would be not using HTTPS, but it is my last chance.

Any suggestion?

**Comment by JC5** [comment]
What did you set `TRUSTED_PROXIES` to?
